src/main/java/com/cryptlex/lexactivator/Release.java

```diff
@@ -16,7 +16,7 @@ public class Release {
     public String[] platforms;
     public ReleaseFile[] files;
 
-    public class ReleaseFile {
+    public static class ReleaseFile {
```

Comment:
why is this static?

Reply:
Since the nested class were crashing while deserialization. I explained the scenario to ChatGPT. Here is what it says:

When Jackson performs deserialization, it relies on the presence of a default constructor to create an instance of the class. By default, if a nested class does not have a non-private default constructor, Jackson may encounter difficulties creating an instance and throw an exception. 

Then it gave three ways to avoid such errors:

One of them is declaring the nested class as static

Reply:
I think while deserialization jackson may require instantiating the Nested class and static keyword allow it to do so without needing the main class.